README.md

```diff
@@ -55,16 +55,17 @@
 
 ### Key Features <a name="key-features"></a>
 
-- **Add new books: Enter the book title and author to add a new book to the list.**
-- **Remove books: Remove books from the list when no longer needed**
-- **Save and remove books from localStorage**
+- **Fetch data from an API.**
+- **Render fetched data.**
+- **Update UI based on the store state.**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- Live Demo -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-<a href="https://microhass.github.io/awesome-books/"> See Project 🚀</a>
+
+<a href="https://microhass.github.io/javascript-capstone-2/dist/"> See Project 🚀</a>
 
```

Comment:
- [ ] Kindly consider recording a video as part of this project requirement, and provide a working link, this one is giving me a 404 error message.

![video](https://github.com/microhass/javascript-capstone-2/assets/53124548/d1ca244c-abae-42eb-987c-f21a0cc76aed)